sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Config/ServiceBusClientFactory.cs

```diff
@@ -14,7 +14,7 @@
 
 namespace Microsoft.Azure.WebJobs.Extensions.ServiceBus.Config
 {
-    internal class ServiceBusClientFactory
+    public class ServiceBusClientFactory
```

Comment:
For any new public types/methods, we need to have good xml doc.

Reply:
Keeping the ServiceBusClientFactory internal, as discussed with @JoshLove-msft offline